cypress/integration/GlossaryAppIntegration.feature

```diff
@@ -0,0 +1,64 @@
+Feature: Glossary app is integrated withing CGOV platform
+
+
+    Scenario: User is able to access a default (or “landing”) page that displays all of the expected Dictionary elements for the Dictionary of Cancer terms.
```

Comment:
So this is conflating two different things, you are combining the testing of the cancer.gov utility bar with the ability for the app module to display. These should be two separate scenarios. This scenarion should just have the first when of:
`When the user is viewing the url "/publications/.../dictionary"`

Reply:
The same goes for spanish, but with spanish urls.